rfcs/2021-12-20-10517-llvm-backend-for-vrl.md

```diff
@@ -649,6 +649,34 @@ to
 We use the same strategy that is used in our Vector binary, which currently uses
 the default "unwind".
 
+### Testing Strategy
```

Comment:
The part I'm still missing here is what we _can't_ capture (reliably) in tests, and thus need documentation/review processes for.

Reply:
Added in 7e1b77b78a087c5ad16d90021a5e21c46df460c2.